requirements/requirements.txt

```diff
@@ -406,9 +407,9 @@ slack-sdk==3.19.4
     # via -r /awx_devel/requirements/requirements.in
 smmap==5.0.0
     # via gitdb
-social-auth-app-django==5.0.0
+social-auth-app-django==5.4.0
```

Comment:
When I see a .0 z stream (semantic versioning `x.y.z`) I always wonder "Is there a newer .1. or .2 version available?". With semantic versioning a new `y` stream indicates new features or breaking changes. With new features comes new bugs. I guess I'm suspicious of new bugs with `.0` releases.

HOWEVER, in this case if we go to `social-auth-app-django` releases https://github.com/python-social-auth/social-app-django/releases they only have `y` releases.

I guess I'm still suspicious of this projects maturity BUT if I zoom out, this project looks like it's sort of a wrapper to djangoize another library. So I guess the lack of bug fix releases is justified.

Not asking for any action here, just brain dumping how I consider dependency updates.

Reply:
It does come from ansible_base. @AlanCoding before upgrading the CI was failing on a check_migrations call because the models use a change that happens in the upgraded version of social_auth_app_django. That hanging migration might not affect functionality for what you need though.

Reply:
That makes sense to me, I would prefer to just go with the newer library version then.